controller/handler_edge_ctrl/common_tunnel.go

```diff
@@ -180,7 +180,9 @@ func (self *baseTunnelRequestContext) ensureSessionForService(sessionId, session
 		self.session, err = self.handler.getAppEnv().Handlers.Session.Read(id)
 		if err != nil {
 			err = internalError(err)
+			return
 		}
+		self.newSession = true
```

Comment:
`newApiSession` ?

Reply:
It's marking that a new session was created in ensureSessionForService. ApiSessions are created in ensureApiSession

Reply:
I didn't unfold the code enough.

Reply:
yeah, with the default context it looks api session related